jetty-core/jetty-util/src/main/java/org/eclipse/jetty/util/thread/AutoLock.java

```diff
@@ -46,8 +48,24 @@ public AutoLock lock()
     }
 
     /**
-     * @see ReentrantLock#isHeldByCurrentThread()
+     * <p>Tries to acquire the lock.</p>
+     * <p>Whether the lock was acquired can be tested
+     * with {@link #isHeldByCurrentThread()}.</p>
+     * <p>Typical usage of this method is in {@code toString()},
+     * to avoid deadlocks when the implementation needs to lock
+     * to retrieve a consistent state to produce the string.</p>
+     *
+     * @return this AutoLock for unlocking
+     */
```

Comment:
```suggestion
     * <p>Tries to acquire the lock.</p>
     * <p>Whether the lock was acquired can be tested
     * with {@link #isHeldByCurrentThread()}.</p>
     * <p>Typical usage of this method is in {@code toString()},
     * to avoid deadlocks when the implementation needs to lock
     * to retrieve a consistent state to produce the string.</p>
     * @see ReentrantLock#isHeldByCurrentThread()
     * @return this AutoLock for unlocking
     */
```